packages/suite/src/views/settings/SettingsDevice/ChangeLanguage.tsx

```diff
@@ -33,6 +41,16 @@ export const ChangeLanguage = () => {
         option => option.value === device?.features?.language,
     );
 
+    const firmwareVersion = getFirmwareVersion(device);
+
+    const isSupportedDevice = device.features?.internal_model !== DeviceModelInternal.T1B1;
+
+    if (!isSupportedDevice) {
+        return null;
+    }
+
+    const doesFirmwareSupportLanguages = isNewerOrEqual(firmwareVersion, [2, 6, 5]);
```

Comment:
I'd approach this a bit differently:

1. Let's do the check in `DeviceSettings` as for the other sections.
2. Let's not display the section at all if the feature is not supported due to outdated firmware - I don't think it brings any value when disabled.
3. Let's use `selectedDevice.availableTranslations` instead of checking version.